package.json

```diff
@@ -47,7 +47,7 @@
     "codelyzer": "^5.0.0",
     "jasmine-core": "^3.4.0",
     "jasmine-spec-reporter": "~4.2.1",
-    "karma": "~4.1.0",
```

Comment:
Revert this change, the rest looks good.

Reply:
x2

Reply:
done!